templates/web/base/report/_main.html

```diff
@@ -8,19 +8,25 @@
 <div class="problem-header clearfix" problem-id="[% problem.id %]">
 
 [% IF c.user.has_permission_to('planned_reports', problem.bodies_str) %]
-<form method="post" action="/my/planned/change" id="planned_form">
+<form method="post" action="/my/planned/change" id="planned_form" class="hidden-label-target">
     <input type="hidden" name="id" value="[% problem.id %]">
     <input type="hidden" name="token" value="[% csrf_token %]">
     <input type="hidden" name="change" value="[% IF c.user.is_planned_report(problem) %]remove[% ELSE %]add[% END %]">
-    <p><input type="submit"
-        data-remove="[% loc('Remove from planned reports') %]" data-add="[% loc('Add to planned reports') %]"
-        value="
+    <p><input
+        type="submit"
+        id="shortlist-report"
+        data-label-remove="[% loc('Remove from shortlist') %]"
+        data-label-add="[% loc('Add to shortlist') %]"
+        data-value-remove="[% loc('Shortlisted') %]"
+        data-value-add="[% loc('Shortlist') %]"
     [%~ IF c.user.is_planned_report(problem) ~%]
-        [% loc('Remove from planned reports') %]
+        value="[% loc('Shortlisted') %]"
```

Comment:
> Should shortlisted look pushed in, like the radio segmented controls do, perhaps?
  
Ooh that’s an interesting idea!

> Then also Inspect could perhaps stay visible, also pushed in, when the inspect part is being shown, and clicking it could come out of inspect mode? Though might worry about losing info then I guess.

We already have a "Cancel" button in the Inspection form, that just links back to `/report/:id`, losing any changes you’ve made in the form. Maybe a pressed-in button labelled "Inspect" is a little more dangerous, because it isn't labelled with a danger word.